docker/compose.yaml

```diff
@@ -131,7 +168,8 @@ services:
       - "host.docker.internal:host-gateway"
```

Comment:
I had to remove the extra_hosts part so that the proxy works correctly.
With these part, I get a 502 Bad Gateway when I call http://localhost/api/v1/about in the browser